rllib/evaluation/rollout_worker.py

```diff
@@ -1,8 +1,10 @@
 from collections import defaultdict
 import copy
+import importlib.util
```

Comment:
Main changes in `RolloutWorker`.
* Dramatically reduced complexity of its ctor signature (due to the fact that most args should have already been defined in the config anyways).
* Now uses AlgorithmConfig under the hood (passing in a dict is still supported, though).
* Moved some of the "conversion" logic that it used to perform on the config into AlgorithmConfig itself (e.g. the construction and unification of the `policies` dict in a multi-agent setup).